### DISCUSSION TITLE
TODO - Some changes in DDEV

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3780] [Ideas]
@sreichel

We need to review this file **/docs/DDEV.md**. In the **Setup cronjob** section the command should be changed to 

```ddev get ddev/ddev-cron```

After installation you will notice that the **.ddev/config.cron.yaml** file no longer exists. According to documentation from here https://github.com/ddev/ddev-cron the configuration is done differently now. 

For the moment I will insert the following configuration in .ddev/config.yaml to see what happens

```
hooks:
  post-start:
    - exec: printf "SHELL=/bin/bash\n* * * * * /var/www/html/cron.sh\n" | crontab
```

In addition we should also add information about the two commands **openmage-install** and **openmage-admin** that we added, extremely useful.

**Comment by sreichel** [comment]
Ill wait for https://github.com/OpenMage/magento-lts/pull/3659.

**Comment by sreichel** [comment]
Quick test ... run ...

```
ddev get ddev/ddev-cron
echo "* * * * * /var/www/html/cron.sh" > .ddev/web-build/openmage.cron
ddev restart
```

If it works, it could be moved to the install command ....

**Reply by addison74** [reply]
In this case we have to choose between creating the command openmage-cron or approving PR #3787. 

We can keep them both too.

**Comment by sreichel** [comment]
Had to be ... for eof ...
```
echo "* * * * * /var/www/html/cron.sh\n" > .ddev/web-build/openmage.cron
```
